automation/src/test/java/org/greenplum/pxf/automation/features/columnprojection/ColumnProjectionTest.java

```diff
@@ -54,7 +55,91 @@ public void checkColumnProjection() throws Exception {
         // SELECT t0, colprojvalue FROM test_column_projection GROUP BY t0, colprojvalue HAVING AVG(a1) < 5 ORDER BY t0;
         // SELECT b.value, a.colprojvalue FROM test_column_projection a JOIN t0_values b ON a.t0 = b.key;
         if (gpdb.getVersion() >= 7) {
-            runTincTest("pxf.features.columnprojection.checkColumnProjection_gp7.runTest");
+            /** The below query (mentioned in above comment as well) is propagating for FDW but not for external-table,
```

Comment:
i believe double ** is used for java docs, but here we are just writing a comment so we can just use one *
```suggestion
            /* The below query (mentioned in above comment as well) is propagating for FDW but not for external-table,